index.html

```diff
@@ -18,5 +18,62 @@
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+      <nav class="nav_bar">
+        <span class="intro">My Blog</span>
+        <ul class="menu">
+          <li class="list"><a class="link" href="#">Home</a></li>
```

Comment:
be a little careful of naming classes "link". It is very generic and not very descriptive. Here "navbar-link" would have been slightly better.